### PR TITLE
fix(Tab): make index.js exports match index.d.ts exports

### PR DESCRIPTION
**Pull request by erautenberg**
## Description

Making the index.js and index.d.ts files better match for `components/index.js` and `TabBar/index.js`

## Checklist

- [ ] all commented code has been removed
- [ ] any new console issues have been resolved
- [ ] code linter and formatter has been run
- [ ] test coverage meets repo requirements
- [ ] PR name matches the expected semantic-commit syntax

### CODE REVIEW

**Comment by soumyaloka** [comment]
Emily, I noticed most of the import statements in `index.js` include the .js extension, while others do not. For the consistency, I suggest we drop the .js extension from all import statements or vice versa which ever works for you.

File: packages/@lightningjs/ui-components/src/components/TabBar/index.js [hunk 1]
```diff
@@ -16,4 +16,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export { default as default } from './TabBar.js';
+import TabBar from './TabBar';
+import Tab from './Tab';
+
+export { TabBar as default, Tab };
```
Comment (THoj13) [inline packages/@lightningjs/ui-components/src/components/TabBar/index.js:22]: TabBar.js also exports Tab. That can probably be removed with this change, but also won't break anything if it remains there too.

**Review by THoj13 — APPROVED** [review]
I left a comment about another optional change, but this looks good to me.

File: packages/@lightningjs/ui-components/src/components/TabBar/index.js (hunk 1 above)
Comment (anthony9187) [reply packages/@lightningjs/ui-components/src/components/TabBar/index.js:22]: I think it'd be best to remove it, this is how we wind up with weird circular dependencies 😬

Comment (erautenberg) [reply packages/@lightningjs/ui-components/src/components/TabBar/index.js:22]: Oh I didn't realize that 🤔 it seems a little odd to me. Thoughts on removing that and just having it live here in the index file?

Comment (anthony9187) [reply packages/@lightningjs/ui-components/src/components/TabBar/index.js:22]: I think it makes the most sense to remove the Tab export from TabBar and have index.js handle all the exports of the module

**Review by anthony9187 — APPROVED** [review]

**Pull request merged**